AUTHORS.rst

```diff
@@ -4,7 +4,7 @@ People
 ------
 
 This work is made available by a community of people, amongst which
-the `INRIA Parietal Project Team <https://parietal.saclay.inria.fr/>`_
+the `INRIA Parietal Project Team <https://team.inria.fr/parietal/>`_
```

Comment:
I guess you want both an author page on nilearn github home page and one in the doc website.

Small technicality: one is actually copied into the other during the sphinx generation, never remember which one.